src/hooks/useENSRegistrationForm.ts

```diff
@@ -238,46 +252,17 @@ export default function useENSRegistrationForm({
 
   const submit = useCallback(
     async submitFn => {
-      const errors = Object.entries(textRecordFields).reduce(
-        (currentErrors, [key, { validations }]) => {
-          const value = values[key as ENS_RECORDS];
-          if (validations?.onSubmit?.match) {
-            const { value: regex, message } =
-              validations?.onSubmit?.match || {};
-            if (regex && value && !value.match(regex)) {
-              return {
-                ...currentErrors,
-                [key]: message,
-              };
-            }
-          }
```

Comment:
wondering about this change, we were enforcing some characters with this but now we'll show an alert. For example if you were typing a space in an username input, the space would get removed automatically.
Just pointing this out to make sure is what we want

Reply:
yeah it was intentional, i felt that it would be overwhelming if your input was getting validated + filtered both at the same time. open to changing it tho